tailwind.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
         peach: "#F4EAE1",
         "peach-light": "#FFF6EF",
         "peach-hover": "#FFE0B2",
+        "pastel-yellow": "#FFC554",
```

Comment:
<img width="677" alt="Screenshot 2021-09-16 at 9 56 28 AM" src="https://user-images.githubusercontent.com/13185699/133549296-c6e5cd34-5028-4349-a3c6-5e7a22004de4.png">
why not use one of these 2 existing colors in tailwind? Looks almost exactly the same. I am sure Karn just moved some color slider to pick this color but it would not make any difference if we chose any of these 2 colors - would avoid having to add custom stuff.